CONTRIBUTORS.md

```diff
@@ -6,5 +6,6 @@ This file lists the contributors who have completed the frontend challenges from
 | Name | GitHub Profile | Challenge |
 | --- | --- | --- |
 | John Doe | [johndoe] | Profile card component |
+| John Doe | [Shiba-Tatsuya](https://github.com/Shiba-Tatsuya) | Profile card component |
```

Comment:
Replace John Doe with your name
Replace Profile card component with your name also

Reply:
sir I changed contributors.md according to you please check